### DISCUSSION TITLE
How can I remove an lsp that is in the ensure_installed list?

### DISCUSSION THREAD

**Discussion by Raagh** [LazyVim/LazyVim#2773] [Q&A] [answered]
Hey so I have the eslint extras installed, which installs eslint_lsp. I want to remove that from the list but leave the rest intact.

I found how to extend lists and how to override the entire list of ensure_installed but I dont want that. I just want to remove one

I cant just uninstall it with mason because it will be installed again when I open neovim

**Comment by dpetka2001** [comment]
That extra only configures the `eslint` LSP server. If you don't want the `eslint` server, then why do you enable the Extra?

**Reply by Raagh** [reply]
I guess I am mixing things. I added this recipe 

```
{
  "neovim/nvim-lspconfig",
  opts = {
    servers = { eslint = {} },
    setup = {
      eslint = function()
        require("lazyvim.util").lsp.on_attach(function(client)
          if client.name == "eslint" then
            client.server_capabilities.documentFormattingProvider = true
          elseif client.name == "tsserver" then
            client.server_capabilities.documentFormattingProvider = false
          end
        end)
      end,
    },
  },
}
```

So I am interested in using the lsp server to make this formatting. But the eslint lsp is consuming 4gb of ram, I cant justify it only for that. (btw how much does it consume for you?)

I guess I can remove the lsp server and just use prettier alone... Not sure if I will get the same result

**Reply by dpetka2001** [reply, marked as answer]
No idea, I don't use `eslint` myself. But that extra you mentioned only configures `eslint` LSP server. So, if you don't want `eslint`, it doesn't make sense to have that Extra enabled.
